.github/workflows/ci.yml

```
@@ -0,0 +1,244 @@
+name: "Continuous Integration"
+on:
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
```

Pourquoi ne pas plutot faire 
```suggestion
  push:
    branches: [master]
  pull_request:
```

---

Très juste